packages/core/src/utils.js

```diff
@@ -528,7 +529,10 @@ export function isSelect(_schema, rootSchema = {}) {
   return false;
 }
 
-export function isMultiSelect(schema, rootSchema = {}) {
+export function isMultiSelect(schema, uiSchema, rootSchema = {}) {
```

Comment:
Why are we restricting this to only use the "select" widget? Can we remove that restriction and leave it up to the user. The reason being, it could be possible, that the customWidget for `select` handles only single select and one could have a separate widget for multi select.